.github/workflows/pipeline-deploy-app-preprod-prod.yaml

```diff
@@ -0,0 +1,125 @@
+name: Pipeline Deployment Application production & pre-production Environment
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: "Select the Tag to deploy artefact from"
+        required: true
+        type: string
+      environment:
+        description: "Select the deployment environment"
+        required: true
+        type: string
+        options:
+          - preprod
+          - prod
+jobs:
+  metadata:
+    name: "Get Metadata"
+    uses: NHSDigital/uec-dos-management/.github/workflows/metadata.yaml@latest
+
+  derive-artefact-address:
+    name: "Get artefact location"
+    uses: NHSDigital/uec-dos-management/.github/workflows/derive-artefact-address.yaml@latest
+    with:
+      tag: ${{ inputs.tag }}
+
+  quality-check:
```

Comment:
Why are we running the quality checks here ? I thought these were done on push to branch as a sort of gatekeeper - ie to ensure quality of stuff before it gets to main. Not sure we need to repeat it here and it would presumably make deployment slower

Reply:
Removed quality check